pennylane/operation.py

```diff
@@ -229,8 +230,8 @@ class Operator(abc.ABC):
         params (tuple[float, int, array, Variable]): operator parameters
 
     Keyword Args:
-        wires (Sequence[int]): Subsystems it acts on. If not given, args[-1]
-            is interpreted as wires.
+        wires (Iterable or Wires): Iterable containing representations of the wires that the operator acts on,
+            or Wires object. If not given, args[-1] is interpreted as wires.
```

Comment:
Technically, this argument can be ``Iterable[Number, str] or Number or str or Wires``. I was trying forth and back to get a concise but truthful description...Shall I add more detail?